src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -159,14 +160,19 @@
 (rf/reg-event-fx
  :wallet/create-keypair-with-account
  (fn [{db :db} [password account-preferences]]
-   (let [{:keys [keypair-name
+   (let [{:keys [workflow
```

Comment:
nit: I'm not so sure about us using the concept of workflow here. 
I've seen in other parts of this too. 
Anyone have any suggestions which might be more applicable? 
Perhaps user-flow? 🤔 
cc @seanstrom @vkjr 

anyway, no need to change it for now. Just something for us to consider for later 👍

Reply:
The reason for appearance "workflow" is that one screen is used to create an account both through a private key and through a random phrase. 
The screen is the same, but the events, data and where they are stored are different